database/index.js

```diff
@@ -1,12 +1,23 @@
 const mongoose = require('mongoose');
+const helper = require('./generateListings');
```

Comment:
I think that you'd better change `helper` to more specific name.

Reply:
Changed `helper` to `seedDB`